scrapers/scraper.py

```
@@ -0,0 +1,424 @@
+def get_tower_abbreviation(x):
+    return ''.join([word[0].lower() if word[0].islower() else word[0].upper() for word in x.replace('-', ' ').split()]).lower()
+
+def open_files(x):
+    global file, file2, info_file
+    file = open(f"public/static/data/games/{x}.txt", "w")
+    file2 = open(f"../api/data/{x}.txt", "w")
+    
+    if x == "etoh":
+        info_file = open("../bot/info.txt", "w")
+
+def init(gname, name, urlss, b_areas, b_sections, b_towers, e_areas=[], e_towers=[], m_types=[], autoo=[False, ""]):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)